fictadvisor-api/src/v2/utils/documentation/telegramGroup/create.ts

```diff
@@ -0,0 +1,39 @@
+import { ApiDocumentationParams } from '../decorators';
+import { TelegramGroupResponse } from '@fictadvisor/utils/responses';
+import { DefaultUnauthorizedResponse } from '../defaultResponses';
+
+export const TelegramGroupDocumentationCreateTgGroup : ApiDocumentationParams = {
+  isAuth: true,
```

Comment:
name of the documentation object must match such format: `{EntityName}Documentation{MethodName}`, `TelegramGroupDocumentationCreate` in this case